# One sample data of each active marker

# Marker identifier
int32 id
# Marker quality number
float32 cond

# 3D coordinates of the marker
geometry_msgs/Point point
